stable/shield/values.yaml

```diff
@@ -0,0 +1,78 @@
+shield-app:
+  enabled: true
+  image:
+    repository: odpf/shield
+    pullPolicy: Always
+    tag: latest
```

Comment:
Do we want to use `latest` or just remove (or comment) and use appVersion from `Chart.yaml` ?

Reply:
Changing it to `0.2.0-rc` as that's the latest image available at the moment.